app/assets/stylesheets/main.sass.scss

```diff
@@ -6,3 +6,27 @@ $govuk-new-link-styles: true;
 .app-\!-inherit-colour {
   color: inherit;
 }
+
+.x-govuk-related-navigation {
+  border-top: 2px solid #1d70b8;
+}
+
+.x-govuk-related-navigation__nav-section {
+  margin-bottom: 30px;
+}
+
+.x-govuk-related-navigation__main-heading {
+  margin-bottom: 10px;
+  margin-top: 15px;
+}
+
+.x-govuk-related-navigation__link-list {
+  list-style: none;
+  margin-bottom: 1.25em;
+  margin: 0;
+  padding: 0;
+}
+
+.x-govuk-related-navigation__link {
+  margin-top: 15px;
+}
```

Comment:
Let's prefix this with `app-` which is the conventional name when declaring app specific CSS classes, so they don't clash with things from the design system or third party libs (the related navigation is part of [x-govuk](https://x-govuk.github.io/govuk-prototype-components/related-navigation/)). 

We can also use nesting and variables to make the code a bit drier, and steal some more styles (like tables ones) from the source of the original component: https://github.com/x-govuk/govuk-prototype-components/blob/main/x-govuk/components/related-navigation/_related-navigation.scss

It should look like this all said and done, but can you double check that the `$govuk-brand-colour` and other variables are correctly being imported from the design system and that this code actually works? :smile: 

```scss
.app-govuk-related-navigation {
  @include govuk-text-colour;
  border-top: 2px solid $govuk-brand-colour;
  
  &__nav-section {
    margin-bottom: govuk-spacing(6);
  }
  
  &__main-heading {
    margin-bottom: govuk-spacing(2);
    margin-top: govuk-spacing(3);
  }
  
  &__link-list {
    list-style: none;
    margin-bottom: 1.25em;
    margin: 0;
    padding: 0;
  }
  
  &__link {
    @include govuk-font($size: 16, $weight: regular, $line-height: 1.45);
    margin-top: govuk-spacing(3);

    @include govuk-media-query($from: tablet) {
      line-height: 1.28;
    }
  }
}
```

Reply:
Done.